agent/engine/docker_task_engine.go

```diff
@@ -490,6 +504,16 @@ func (engine *DockerTaskEngine) AddTask(task *api.Task) error {
 		return nil
 	}
 
+	// Start task that is waitng for the execution role credentials
+	_, ok := engine.managedTasks[task.Arn]
+	if !ok {
+		existingTask.SetDesiredStatus(task.GetDesiredStatus())
+		existingTask.SetExecutionRoleCredentialsID(task.GetExecutionCredentialsID())
+		seelog.Infof("engine: task received updated execution credentials, task: %s", task.String())
```

Comment:
Same as above. Log message should begin with upper case letter. Please fix everywhere. Also, just logging task Arn should be enough here.